src/ui/Breadcrumb/Breadcrumb.ts

```diff
@@ -147,6 +156,17 @@ export class Breadcrumb extends Component {
     this.bind.onRootElement(QueryEvents.deferredQuerySuccess, () => this.handleDeferredQuerySuccess());
     this.bind.onRootElement(QueryEvents.queryError, () => this.handleQueryError());
   }
+
+  private focusFirstEnabledResultList() {
+    const resultLists = this.searchInterface.getComponents<Component>('ResultList');
+    const firstEnabledResultList = find(resultLists, resultList => resultList.disabled === false);
+    if (firstEnabledResultList) {
+      // Problem with types definition of TypeScript 2.8
+      // default .focus() definition is incomplete.
+      // force cast to any
+      (firstEnabledResultList.element as any).focus({ preventScroll: true });
```

Comment:
I am unable to reproduce this error on my side. I removed the `as any` cast without getting an error, then I upgraded to `typescript@2.8.1` (same version as in the `package.json`) and still had no error. After the upgrade, I see the `focus` function's definition as being part of `HTMLOrSVGElement`:
![image](https://user-images.githubusercontent.com/54454747/91874770-a186fc00-ec48-11ea-890d-0c80de8f087c.png)

Is anybody else experiencing the error?